python/pyhail/dataset.py

```diff
@@ -1620,10 +1876,96 @@ def tdt(self, fam, root='va.tdt'):
         """Find transmitted and untransmitted variants; count per variant and
         nuclear family.
 
+        **Examples**
+
+        Compute TDT association results
+
+        >>> vds.tdt("data/sample.fam").export_variants("Variant = v, va.tdt.*")
```

Comment:
I like putting different commands on separate lines.  (This applies in a few other places.)